src/pytest_tap/plugin.py

```diff
@@ -36,13 +36,17 @@ def __init__(self, config: pytest.Config) -> None:
     def pytest_runtestloop(self, session):
         """Output the plan line first."""
         option = session.config.option
-        if option.tap_stream or option.tap_combined:
+        if (option.tap_stream or option.tap_combined) and not (
+            session.config.pluginmanager.has_plugin("subtests")
+        ):
```

Comment:
I don't understand this logic. Can you help me understand why we don't want to write the plan when the subtests plugin is available?